wasi-common/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "wasi-common"
+version.workspace = true
+authors.workspace = true
+description = "WASI implementation in Rust"
+license = "Apache-2.0 WITH LLVM-exception"
+categories = ["wasm"]
+keywords = ["webassembly", "wasm"]
+repository = "https://github.com/bytecodealliance/wasmtime"
+readme = "README.md"
+edition.workspace = true
+include = ["src/**/*", "WASI/phases/**/*", "README.md", "LICENSE", "build.rs"]
+build = "build.rs"
```

Comment:
Let's add `publish=false` to make sure we never accidentally overwrite the "real" wasi-common